fvm/transactionInvoker.go

```diff
@@ -126,18 +126,23 @@ func (i *TransactionInvoker) Process(
 		}
 
 		location := common.TransactionLocation(proc.ID[:])
-
-		err := vm.Runtime.ExecuteTransaction(
-			runtime.Script{
-				Source:    proc.Transaction.Script,
-				Arguments: proc.Transaction.Arguments,
-			},
-			runtime.Context{
-				Interface:         env,
-				Location:          location,
-				PredeclaredValues: predeclaredValues,
-			},
-		)
+		var err error
+		// disable this transaction as it eats through memory
+		if txIDStr == "51be45d5d298403e7798e5c8048102529acae4b05e16f795220f2ac1034ccf64" {
+			txError = errors.NewLedgerIntractionLimitExceededError(state.DefaultMaxInteractionSize, state.DefaultMaxInteractionSize)
```

Comment:
We might want to add a dedicated error here, as this isn't a "ledger interaction limit exceeded" error